css/style.css

```diff
@@ -9,6 +9,8 @@ body {
     margin:0 auto;
     background:#fff;
     max-width: 100%;
+    padding-right: 15px;
+    padding-left: 15px;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Using width with padding-left can sometimes make elements larger than you expect.](https://www.codacy.com/app/naczelnik/ambasadorzy-siecobywatelska-pl/file/1624604468/issues/source?bid=2991913&fileBranchId=3033149#l13)